modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledNestedMenuItemTest.ts

```diff
@@ -11,13 +12,19 @@ describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledNestedMenuItemT
   const codeMenuItemSelector = '[role="menuitem"]:contains("Code")';
   const preferencesMenuItemSelector = '[title="Preferences"]';
   const servicesMenuItemSelector = '[title="Services"]';
+  const rightArrowSelector = '.tox-collection__item-caret svg';
+  const disabledColour = 'rgba(34, 47, 62, 0.5)';
```

Comment:
This doesn't work on IE 11 as it'll use hex color codes instead of RGB. In saying that though, this is not something we should be including in this test because that's related to the oxide styles. All this should be testing is the class is there.

- [ ] Remove tests that check the svg fill computed color

Reply:
I'm checking the class `tox-collection__item--state-disabled` now.